src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/Http/FrameOfT.cs

```diff
@@ -95,6 +96,8 @@ public Frame(IHttpApplication<TContext> application, FrameContext frameContext)
                         {
                             try
                             {
+                                KestrelEventSource.Log.RequestStart(this);
```

Comment:
I wonder if we need these events. Hosting does the exact same thing what value do we add here?